src/NHibernate.Test/Linq/MiscellaneousTextFixture.cs

```diff
@@ -27,7 +27,8 @@ public void WhereUsingGuidProperty()
         [Test(Description = "This sample uses Count to find the number of Orders placed before yesterday in the database.")]
         public void CountWithWhereClause()
         {
-            var q = from o in db.Orders where o.OrderDate <= DateTime.Today.AddDays(-1) select o;
+            var yesterday = DateTime.Today.AddDays(-1);
+            var q = from o in db.Orders where o.OrderDate <= yesterday select o;
```

Comment:
Here is an example of breaking change.
Previously, `DateTime.Today` was pre-evaluated and not translated to SQL, and `AddDays` too. But now, `DateTime.Today` is no more pre-evaluated, forcing `AddDays` to be translated. But there is no generator for it, so the query fails, unless moving all this into a local variable.